scopes/mdx/ui/snippet/snippet.tsx

```diff
@@ -14,9 +15,9 @@ export function Snippet({ children, live }: SnippetProps) {
   const scope = Object.assign({}, components, {
     mdx,
   });
-
-  if (live) {
-    return <Playground code={children} scope={scope} />;
-  }
-  return <CodeSnippet>{children}</CodeSnippet>;
+  return (
```

Comment:
can't pass class to CodeSnipper / Playground? what about {...rest}

Reply:
playground doesnt get a className because I cant pass it to `LiveProvider`